components/Profile/Edit.js

```diff
@@ -0,0 +1,223 @@
+import React, { Fragment } from 'react'
+import { graphql, compose } from 'react-apollo'
+import gql from 'graphql-tag'
+import { css } from 'glamor'
+
+import withT from '../../lib/withT'
+import withMe from '../../lib/apollo/withMe'
+
+import { errorToString } from '../../lib/utils/errors'
+
+import {
+  InlineSpinner,
+  Button,
+  linkRule,
+  colors
+} from '@project-r/styleguide'
+
+const styles = {
+  editLink: css({
+    display: 'block',
+    marginTop: 5
+  })
+}
+
+const EditLink = ({children, onClick, ...props}) =>
+  <a
+    href='#'
+    onClick={(e) => {
+      e.preventDefault()
+      onClick(e)
+    }}
+    {...props}
+    {...linkRule}
+    {...styles.editLink}>
+    {children}
+  </a>
+
+const Edit = ({me, user, t, state, setState, startEditing, update}) => {
```

Comment:
No action required, but I think in the long run we'd benefit from documenting which of `state, setState, startEditing, update` is an object, a function or a bool (no wait, startEditing is a function, too). OK to leave that to more quiet times though ;)

Reply:
added prop types

Reply:
👍